packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -510,7 +511,18 @@ class NetworkImage extends ImageProvider<NetworkImage> {
     );
   }
 
-  static final HttpClient _httpClient = HttpClient();
+  /// Do not access this field directly; use [_httpClient] instead.
```

Comment:
nit: `//` instead of `///` since this comment of a private member doesn't have public doc quality.

Reply:
I had done that so that refactoring tools would see the reference to the `_httpClient` variable.  But I agree it's weird - I changed it to `//`